examples/with-pnpm-workspaces/packages/web-app/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

should we enable Typescript strict mode?
```suggestion
    "strict": true,
```